launch/launch/utilities/typing_file_path.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Rover Robotics, c/o Dan Rose
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import typing
+
+# Type of a filesystem path
+FilePath = typing.TypeVar('FilePath', str, bytes, os.PathLike)
```

Comment:
A separate file for a single type which also only used in a single file once seems unnecessary to me.

Reply:
This comment is still pending.

Reply:
Files are cheap and there is not a better place to put it. This code is written as intended.